Adafruit_GFX.cpp

```diff
@@ -71,596 +71,680 @@ POSSIBILITY OF SUCH DAMAGE.
 #endif
 
 Adafruit_GFX::Adafruit_GFX(int16_t w, int16_t h):
-  WIDTH(w), HEIGHT(h)
```

Comment:
not really, it's more of auto-spacing of the editor. There were many mixed spaces/tabs and that is not nice to look at and read. This diff is best to be looked at as result rather than diff. Some methods are moved and some are added. Git diff is confusing sometimes.